.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## SUMMARY:
+
+## GITHUB ISSUE (Open Source Contributors)
+
+## JIRA TASK (Eightfold Employees Only):
+
+## CHANGE TYPE:
+
+-   [ ] Bugfix Pull Request
+-   [ ] Feature Pull Request
+
+## TEST COVERAGE:
+
+-   [ ] Tests for this change already exist
+-   [ ] I have added unittests for this change
+
+## TEST PLAN:
```

Comment:
Any reason why we need this ?

Reply:
I've included manual testing as part of the roadmap for each component. Will also help seed the project if we provide more details per task